userguide/tutorials/user_management.adoc

```diff
@@ -9,30 +9,40 @@ Each request to Kill Bill requires authentication. The authentication phase veri
 
 Note that the authentication/authorization is orthogonal to the tenant access: The set of permissions associated to a given user is the same for all tenants (for which he has the keys). So, a potential superuser with all the permissions and with access to the `api_key`/`api_secret` for all tenants would have full control of the entire system. It is common practise to create a cross-tenant admin with the ability to only manage tenants and nothing else, and then have one superadmin per tenant.
 
+The full list of available permissions can be found https://github.com/killbill/killbill-api/blob/master/src/main/java/org/killbill/billing/security/Permission.java[here].
 
-= User Management APIs
 
-== Configuration of the User/Role/Permissions
+== User Management APIs
 
+=== Configuration of the User/Role/Permissions
 
-Shiro supports multiple backend implementations, and we integrated 3 of them:
+Shiro supports multiple backend implementations, and we integrated a few of them including:
 
 * Simple configuration file (`shiro.ini` file)
 * Database configuration
 * LDAP integration
+* Okta integration
 
-For simple deployments where only few static users/roles need to be assigned, the `shiro.ini` file is the simplest and the best option. For large entreprise solution, an LDAP integration is probably the most flexible. For cases where LDAP is not available, a good middle ground is to rely on a mixed solution between the `shiro.ini` file and the database backend. This is what the rest of this section will describe:
+For simple deployments where only few static users/roles need to be assigned, the `shiro.ini` file is the simplest and the best option. By default both Kill Bill (`shiro.ini`) and Kaui ship with an `admin` user (and its password `password`) that has all the permissions. One can immediatley log-in to Kaui using such user, and start configuring tenants as needed.
+
+Changing such user (e.g to use `superadmin`) or password would require editing `shiro.ini` on the Kill Bill side to make sure this initial user can log-in to Kaui (see below).
+
+For a large entreprise solution, an LDAP integration is probably the most flexible. For cases where LDAP is not available, a good middle ground is to rely on a mixed solution between the `shiro.ini` file and the database backend. This is what the rest of this section will describe.
```

Comment:
`This is what the rest of this section will describe.` This is confusing because there is nothing left in `Configuration of the User/Role/Permissions` section, and the rest of the document is discussing other things than `shiro.ini` + `database` model.